tests/test_generic_api_client.py

```diff
@@ -4,7 +4,7 @@
 import pytest
 import sys
 
-from rev_ai.models.customer_url_data import CustomerUrlData
+from src.rev_ai.models.customer_url_data import CustomerUrlData
```

Comment:
@aaron-wilson-rev a lot of these paths were giving me import errors in my ide. Seems like tests were passing on your branch though. Not sure why

Reply:
I have the source root in my python path, I'm guessing the CI job does too and maybe you don't?

Reply:
Looks like it does not break anything to have the full path regardless, so probably better to have that anyway